README.md

```diff
@@ -138,31 +132,61 @@ pip install artifacts/tensorflow_recommenders_addons-*.whl
 ```
 
 ##### GPU Support
-**Tensorflow Recommender-Addons** also supports to build on GPU. Please set the `TF_CUDA_VERSION`, `TF_CUDNN_VERSION`,
-`TF_NEED_CUDA`, and `CUDNN_INSTALL_PATH` in environment variables:
-```
-TF_CUDA_VERSION=11.0 TF_CUDNN_VERSION=8 TF_NEED_CUDA=1 CUDNN_INSTALL_PATH="/data/dev/packages/cuda" python configure.py
+Only `TF_NEED_CUDA=1` is required and other environment variables are optional:
+```shell
+TF_NEED_CUDA=1 \
+TF_CUDA_VERSION=11.0 \
+TF_CUDNN_VERSION=8 \
+CUDA_TOOLKIT_PATH="/usr/local/cuda" \
+CUDNN_INSTALL_PATH="/usr/lib/x86_64-linux-gnu" \
+python configure.py
 ```
 And then build the pip package and install:
-```
+```shell
 bazel build --enable_runfiles build_pip_pkg
-bazel-bin/build_pip_pkg artifacts
-pip install artifacts/tensorflow_recommenders_addons-*.whl
-```
+TF_NEED_CUDA=1 bazel-bin/build_pip_pkg artifacts
+pip install artifacts/tensorflow_recommenders_addons_gpu-*.whl
+```
+
+##### Data Type Matrix for `tfra.dynamic_embedding.Variable` 
+|  Values \\ Keys  | int64  | int32 | string |
+|:----:|:----:|:----:|:----:| 
+| float  | CPU, GPU | CPU | CPU |
+| half  | CPU, GPU | - | CPU |
+| int32  | CPU, GPU | CPU | CPU |
+| int8  | CPU, GPU | - | CPU |
+| int64  | CPU | - | CPU |
+| double  | CPU, CPU | CPU |
+| bool  | - | - | CPU |
+| string  | CPU | - | - |
+
+##### To use GPU by `tfra.dynamic_embedding.Variable`
+The `tfra.dynamic_embedding.Variable` will ignore the device placement mechanism of TensorFlow, 
```

Comment:
I think it should also be mentioned in [API annotation](https://github.com/tensorflow/recommenders-addons/blob/master/tensorflow_recommenders_addons/dynamic_embedding/python/ops/dynamic_embedding_variable.py#L164-L165) and [doc](https://github.com/tensorflow/recommenders-addons/blob/master/docs/api_docs/dynamic_embedding/Variable.md)

Reply:
accept